server/fleet/app.go

```diff
@@ -530,6 +557,7 @@ func (c *AppConfig) ApplyDefaultsForNewInstalls() {
 func (c *AppConfig) ApplyDefaults() {
 	c.Features.ApplyDefaults()
 	c.WebhookSettings.Interval.Duration = 24 * time.Hour
+	// TODO: How is this supposed to work? Should we set org_info.contact_url here? It doesn't seem to make a difference.
```

Comment:
IIRC this gets called the first time we create/save the `AppConfig` struct in the database